WTWSMS/decisions/VIET_culture_decisions.txt

```diff
@@ -198,15 +198,18 @@ decisions = {
 			age = 16
 			scaled_wealth = 0.5
 			prisoner = no
-			religion_group = christian
-			NOT = { has_character_flag = VIET_celtic_high_cross }
 			OR = {
-				culture = saxon
+				religion = insular
+				religion = pelagianism
 				AND = {
-					culture_group = celtic
-					NOT = { culture = breton }
+					religion_group = christian
+					OR = {
+						culture = saxon
+						culture = old_saxon
+					}
```

Comment:
I'm not talking about Saxons. Saxons didn't do it anyway.

Celtic Pagans shouldn't use the runestone mechanic. They erected crosses because it became a cultural thing, to my knowledge.